src/scenarios/personalization/ArrayAddFavorites/ArrayAddFavorites.tsx

```diff
@@ -0,0 +1,245 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import {
+  getMoviesQuery,
+  getTvShowsQuery,
+  setArrayItemMutation,
+} from './graphql-documents';
+
+interface entity {
+  id: string;
+  title: string;
+  type: string;
```

Comment:
Since `type` is a string literal, we can have it as:

`type: 'Movie' | 'TV Show';`